### ISSUE TITLE
Duplicate documentDescribes relation when generating documents with the MultiFormatStore

### ISSUE THREAD

**Issue by armintaenzertng** [spdx/spdx-java-jackson-store#37]
```
        ISerializableModelStore modelStore = new MultiFormatStore(new InMemSpdxStore(), MultiFormatStore.Format.XML);

        String documentUri = "https://someuri";
        ModelCopyManager copyManager = new ModelCopyManager();

        SpdxDocument document = SpdxModelFactory.createSpdxDocument(modelStore, documentUri, copyManager);

        document.setSpecVersion(Version.TWO_POINT_THREE_VERSION);
        document.setName("SPDX-tool-test");

        Checksum sha1Checksum = Checksum.create(modelStore, documentUri, ChecksumAlgorithm.SHA1, "d6a770ba38583ed4bb4525bd96e50461655d2758");

        AnyLicenseInfo concludedLicense = LicenseInfoFactory.parseSPDXLicenseString("LGPL-2.0-only OR LicenseRef-2");

        SpdxFile fileA = document.createSpdxFile("SPDXRef-fileA", "./package/fileA.c", concludedLicense,
                        List.of(), "Copyright 2008-2010 John Smith", sha1Checksum)
                .build();

        SpdxFile fileB = document.createSpdxFile("SPDXRef-fileB", "./package/fileB.c", concludedLicense,
                        List.of(), "Copyright 2008-2010 John Smith", sha1Checksum)
                .build();

        document.getDocumentDescribes().addAll(List.of(fileA, fileB));

        modelStore.serialize(doc.getDocumentUri(), new FileOutputStream("output.xml"));
```
The above code produces an SPDX document that contains the following lines:
```
  <documentDescribes>SPDXRef-fileA</documentDescribes>
  <documentDescribes>SPDXRef-fileA</documentDescribes>
  <documentDescribes>SPDXRef-fileB</documentDescribes>
```
i.e. the reference to fileA appears twice. The same happens for the other format types for the MultiFormatStore, but not when using the RdfStore or the TagValueStore.

**Comment by goneall** [comment]
I found a rather suspicious comment in `JacksonSerializer`:

```
//TODO: Remove duplicate relationships
```

**Comment by goneall** [comment]
OK - Think I solved it with PR #36

**Comment by goneall** [comment]
Resolved with PR #36
